osu.Game/Input/Bindings/DatabasedKeyBinding.cs

```diff
@@ -24,11 +27,12 @@ public string KeysString
             private set { KeyCombination = value; }
         }
 
+        [Indexed]
         [Column("Action")]

```

Comment:
I would prefer if you didn't use `new` and instead used a different name such as `DatabasedAction`.